roles/minidlna/tasks/main.yml

```diff
@@ -7,14 +7,26 @@
   blockinfile:
     path: /etc/minidlna.conf
     block: "{{ lookup('template', 'minidlna.conf.j2') }}"
+  register: minidlna_config
 
 - name: add miniDLNA user to NAS group
   user:
     name: minidlna
     groups: "{{ nas.user.name }}"
 
-- name: start
+- name: configure firewall
+  ufw:
```

Comment:
generally better do configure ufw only if feature is enabled (and ufw installed)

Reply:
marked as a future task